puppet/modules/motd/manifests/params.pp

```diff
@@ -1,6 +1,6 @@
 class motd::params {
   case $::osfamily {
-    redhat, debian, suse: {
+    'RedHat', 'Debian', 'Suse': {
```

Comment:
I took this a step further in https://github.com/theforeman/foreman-infra/pull/1310